masterlist.yaml

```diff
@@ -1083,6 +1083,18 @@ plugins:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/2357/'
         name: 'Enhanced Blood Textures SE on Nexus Mods'
     group: *overhaulsGroup
+  - name: 'Particle Patch for ENB SSE.esp'
+    url:
+      - link: 'http://enbseries.enbdev.com/forum/viewtopic.php?t=1499'
+        name: 'Particle Patch for ENB SSE on the ENB Forums'
+    after:
+      - 'Unofficial Skyrim Special Edition Patch.esp'
```

Comment:
It might be correct but It's not needed, `ESM` plugins can't sort after non `ESM` flagged plugins.

Reply:
My particle patch _is_ ESM and ESL tagged, because of ESL-related load order concerns (I need to patch some parts of the patch later in the load order).

If we don't want to worry about that possibility, I'll remove that `after:`.